docs/02-app/02-api-reference/01-components/link.mdx

```diff
@@ -179,6 +179,10 @@ export default function Page() {
 }
 ```
 
+> **Good to know**:
+>
+> - If `{children}` from a Layout is not present in the current viewport, Next.js will instead scroll to the top of the `{children}`.
```

Comment:
```suggestion
> - Next.js will scroll to the part of the page that changes as a result of the navigation, if it's not visible in the viewport.
```

Reply:
I feel like not specifying `{children}` is too vague <picture data-single-emoji=":thinkies:" title=":thinkies:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/thinkies/d164ba479fed9c0d.png" alt=":thinkies:" align="absmiddle"></picture> 

What about `Page` with a link to the [Page](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#pages)?

Reply:
`Next.js will scroll to the [Page](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#pages) if it's not visible in the viewport upon navigation.`